backend/django/api/views.py

```
@@ -1,92 +1,217 @@
+
+        return {
+            "accessToken": validated_data.get("access"),
+            "refreshToken": validated_data.get("refresh"),
+            "user": {"id": user.id, "email": user.email},
+        }
+
+
+class EmailLoginView(APIView):
+    """Login endpoint that issues JWT tokens using email and password."""
+
+    permission_classes = [AllowAny]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)